contracts/Acct.sol

```diff
@@ -35,72 +35,112 @@ contract Acct is Ownable, IAcct {
     }
 
     modifier isUnlocked() {
-        require(block.timestamp >= unlockTime, "Contract is time-locked");
+        // solhint-disable-next-line not-rely-on-time
+        require(block.timestamp >= unlockTime, "Acct: time-locked");
         _;
     }
 
+    // solhint-disable-next-line no-empty-blocks
+    receive() external payable {}
+
     /**
      * @dev Set unlock time
      * @param newUnlockTime Absolute time at which contract is unlocked
      */
     function setUnlockTime(uint256 newUnlockTime) external override onlyOwner isUnlocked {
-        emit LogTimeLock(msg.sender, unlockTime, newUnlockTime);
+        emit LogTimeLock(_msgSender(), unlockTime, newUnlockTime);
         unlockTime = newUnlockTime;
     }
 
     /**
      * @dev Transfer ownership control to NFT registry
      */
     function transferOwnershipToNFT() external override onlyOwner {
-        IOwnerRegistry(REGISTRY_ADDR).mint(address(this));
-        transferOwnership(REGISTRY_ADDR);
+        transferOwnershipToNFT(REGISTRY_ADDR);
+    }
+
+    function transferOwnershipToNFT(address registry) public onlyOwner {
```

Comment:
Lack of the `override` keyword on an external function indicates that `transferOwnershipToNFT` should be added to the interface file.

Reply:
That seems correct. This came from the auditors - I'll review their notes and then make the appropriate changes